kebechet/managers/update/update.py

```diff
@@ -797,16 +805,21 @@ def run(self, labels: list) -> Optional[dict]:
             )
 
             for e in runtime_environments:
-                self.runtime_environment = e
+                self.runtime_environment = e or "default"
```

Comment:
is the default here suppose to be "."?

Reply:
`self.runtime_environment` is used to create issue contents so I opted for something human readable